Automattic/wp-calypso
# State: fixup stored cards schema, stub noisy console warn on test

I missed this in testing #7697 This PR updates the stored-card schema to match the api response.

## Testing Instructions
- Clear IndexedDB
- Set debug to `localStorage.debug = 'calypso:state;'` 
- Make sure your user has a stored credit card
- Navigate to http://calypso.localhost:3000/checkout/:siteId:/personal
- Reload the page
- Check that the initial state contains `storedCard` in
<img width="1111" alt="screen shot 2016-08-29 at 9 41 52 am" src="https://cloud.githubusercontent.com/assets/1270189/18059227/e96068ee-6dcc-11e6-9e69-b5c28fb0015a.png">
- Check that we have no schema validation errors in console
- We can still checkout normally

cc @lamosty 

Test live: https://calypso.live/?branch=fix/stored-card-schema
